packages/dx-react-core/src/plugged/template-placeholder.jsx

```diff
@@ -63,7 +63,7 @@ export class TemplatePlaceholder extends React.Component {
     if (this.template) {
       const { children: templateContent } = this.template;
 
-      content = templateContent();
+      content = templateContent() || null;
```

Comment:
should wo do null or undefined here?

Reply:
There should be `null` because it is used as a return value of the `render()` method. `null` is valid in this case but `undefined` is not.